### DISCUSSION TITLE
GPU temp blinking and don`t cotrol gpu fans

### DISCUSSION THREAD

**Discussion by pwflamy** [seerge/g-helper#2265] [Q&A] [answered]
Hello! 

I have a ROG Strix G17 G713PI laptop. Usually I use Standard mode in G-Helper and when I choose "NV GPU Only" in the NVidia Control Panel it works fine, well, almost. The temperature is shown constantly and it is the same as from the Task Manager. But fans work at 2500 RPM while temp is 42 oC and it doesn't match the gpu fan profile that I setted in G-Helper.

When I choose "Auto select" in the NVidia Control Panel, something goes wrong. First of all, the temperature disappears and appears at times or if I do some action like open Task Manager. The gpu fans totally lose control. They work on 2700 RPM until the temperature appears and it triggers something that fans start to change their speed. For example, now my laptop works on iGPU, the temperature doesn't show and fans speed is 2700 RPM. I open Task Manager, the temperature appears in G-Helper, it is 40 oC and fans are turning off. But only for a few seconds, then temperature disappears in G-Helper, fans begin working at 2700 RPM. If I hide and open the Task Manager window one more time. then it will all happen again. 

<img width="823" alt="image" src="https://github.com/seerge/g-helper/assets/45737605/231f8761-4afb-40ac-8c36-9093e8706d69">

I've already tried to clear reinstall amd and nvidia drivers. Update BIOS to 329.

**Comment by seerge** [comment, marked as answer]
@pwflamy hello, please read the FAQ https://github.com/seerge/g-helper/wiki/FAQ#how-does-g-helper-control-my-fan-speeds

G-Helper DOES NOT CONTROL FANS,

Temperature disappears cause Optimus puts GPU to sleep, but fans work independently from that based on BIOS own sensors

Also don’t expect fan curve to be followed very precisely. It’s being set in some abstract % points and how it will be interpreted by actual BIOS often varies.
